doc/man/man3/crypto_argon2.3monocypher

```diff
@@ -245,6 +264,40 @@ Length of
 in bytes.
 Must be zero if there is no additional data.
 .El
+.Pp
+The arguments in the
+.Fa config
+and
+.Fa extras
+structs may overlap or point at the same buffer.
+.Pp
+Use
+.Xr crypto_verify16 3monocypher ,
+.Xr crypto_verify32 3monocypher ,
+or
+.Xr crypto_verify64 3monocypher
+to compare password hashes to prevent timing attacks.
+.Pp
+To select the
+.Fa nb_blocks
+and
+.Fa nb_passes
+parameters, it should first be decided how long the computation should
+take.
+For user authentication, values somewhere between half a second
+(convenient) and several seconds (paranoid) are recommended.
+The computation should use as much memory as can be spared.
+.Pp
+Since parameter selection depends on your hardware, some trial and error
+will be required in order to determine the ideal settings.
+Three iterations and 100000 blocks
+(one hundred megabytes of memory)
+is a good starting point.
+Adjust
+.Fa nb_blocks
+first.
+If using all available memory is not slow enough, increase
+.Fa nb_passes .
```

Comment:
We may want to revise this section based on current recommendation. I'm thinking of scoobz's advice Soatok outlined in his recent article.

Or we may not. I believe this procedure is still good. The starting point may need to be increased up to 300MB. Perhaps. I'm not sure.

Reply:
What makes you say 300MB? Sc00bz's [website](https://tobtu.com/minimum-password-settings/) allows for much lower as a baseline.

Reply:
I was just throwing that around. I ended up not changing my original numbers. (Well I _did_ add an Argon2id baseline that's 300MB, but that's because we typically use it with only one pass, while Argon2i needs 3).

Now ScOObz is describing _minimum_ settings, and it's not clear for which use case. A login in a low-enough stakes website is certainly cool with that. Your password manager's master password though? probably needs more.

Anyway, apparently I was panicking for no reason, thanks for the link. I can resolve the thread now.